src/main/java/billgates/view/gui/TopMenuBar.java

```diff
@@ -27,16 +37,89 @@ private void initMenu() {
 
         this.add(this.importMenu);
         this.add(Box.createRigidArea(new Dimension(ActionPanel.VERTICAL_GAP, 0)));
-        // importMenu should be disabled  before signing in
-        this.importMenu.setEnabled(false);
-
         this.add(this.settingsMenu);
         this.add(Box.createRigidArea(new Dimension(ActionPanel.VERTICAL_GAP, 0)));
-
         this.add(this.helpMenu);
+
+        // Add function to import menu
+        importMenu.addMenuListener(new MenuAdaptor(){
+            @Override
+            public void menuSelected(MenuEvent e) {
+                importBills();
+            }
+        });
+        this.add(Box.createRigidArea(new Dimension(ActionPanel.VERTICAL_GAP, 0)));
+
+        // Add function to setting menu
+        settingsMenu.addMenuListener(new MenuAdaptor() {
+            @Override
+            public void menuSelected(MenuEvent e) {
+                System.out.println("yes");
+                int[] setting = setting();
+
+                mainFrame.getBillPanel().changeColor(colors[setting[0]]);
+                mainFrame.getActionPanel().changeColor(colors[setting[0]]);
+
+                mainFrame.getActionPanel().changeFont(fonts[setting[1]]);
+                mainFrame.getBillPanel().changeFont(fonts[setting[1]]);
+
+            }
+        });
+    }
+
+    // Function to set up the setting dialog
+    private int[] setting() {
+//        SettingDialog setdlg = mainFrame.getSettingDialog();
+        SettingDialog setdlg = new SettingDialog(mainFrame);
+        int[] setting = new int[2];
+        if (setdlg.exec()) {
+            int colorTargetIndex = setdlg.getMyColor();
+            int fontTargetIndex = setdlg.getMyFont();
+            SettingDialog.setFontField(fontTargetIndex);
+            SettingDialog.setColorField(colorTargetIndex);
+            setting[0] = colorTargetIndex;
+            setting[1] = fontTargetIndex;
+            return setting;
+        }
+        setting[0] = 0;
+        setting[1] = 0;
+        return setting;
     }
 
     public JMenu getImportMenu() {
         return importMenu;
     }
-}
+
+    // Function to choose file in your computer
+    private void importBills() {
+        System.out.println("imported");
+        JFileChooser chooser = new JFileChooser();
+        //JDialog chooserDialog = chooser.createDialog(mainFrame);
+        FileNameExtensionFilter filter = new FileNameExtensionFilter("Your Bills", "csv");
+        chooser.setFileFilter(filter);
+        int ret = chooser.showOpenDialog(mainFrame);
+        if (ret == JFileChooser.APPROVE_OPTION) {
+            File file = chooser.getSelectedFile();
+            loadFile(file);
+        }
+    }
+
+    //Function to load file after loading file
+    public void loadFile(File file) {
+        // CONTINUE!!!
+        System.out.println("loaded");
+    }
+
+    // Create a MenuAdapter to simplify our code
+    public abstract class MenuAdaptor implements MenuListener{
```

Comment:
This may not be the proper way to handle menu events as I mentioned above.

You may check that documentation and then recode the event handling part.